testsuite/features/support/twopence_init.rb

```diff
@@ -143,18 +140,25 @@ def get_target(host)
 # * for the usual clients, it is the full hostname, e.g. hmu-centos.tf.local
 # * for the PXE booted clients, it is derived from the branch name, the hardware type,
 #   and a fingerprint, e.g. example.Intel-Genuine-None-d6df84cca6f478cdafe824e35bbb6e3b
+# rubocop:disable Metrics/MethodLength
+# rubocop:disable Metrics/AbcSize
```

Comment:
Why are we skipping rubocop for these? Were there warnings?

Reply:
yes rubocop was barfing